client/src/pages/Profile.tsx

```
@@ -84,496 +101,486 @@ interface Language {
+										<Skeleton className='h-32 w-full' />
+									</>
+								)}
+							</div>
+
+							{/* GitHub Stats */}
+							<div className='space-y-8'>
+								<h2 className='text-xl font-bold'>GitHub Streak Stats</h2>
+								{streakStat ? (
+									<div
+										className='overflow-x-auto'
+										dangerouslySetInnerHTML={{ __html: streakStat }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)